frontend/ui/src/selectors/conversations.ts

```diff
@@ -3,35 +3,42 @@ import {filter, reverse, sortBy, values} from 'lodash-es';
 import {Conversation} from 'httpclient';
 import {StateModel} from '../reducers';
 import {ConversationMap} from '../reducers/data/conversations';
+import {ConversationRouteProps} from "../pages/Inbox";
+
+export const getCurrentConversation = (state: StateModel, props: ConversationRouteProps) =>
```

Comment:
@ogbeche77 This is what I meant in our call the other week :) This selector allows every component within the inbox page that is connected to the router using `withRouter` to access the currently open conversation in the store.